locale/translation.example.json

```diff
@@ -3,5 +3,15 @@
   "nested": {
     "firstTranslation": "community",
     "secondTranslation": "projects"
+  },
+  "pages": {
```

Comment:
Vím, že toto navazuje na diskusi na Slacku. Ještě nad tím trochu uvažuji, zda klíče do example přidávat nebo ne. Vypadá to spíše, že ano, ale rád bych to provedl cestou přidání všech najednou jako jeden commit spolu s informativním textem do README. Můžeš tedy prozatím prosím tyto změny (klíče a také generované typy) odstranit?